### ISSUE TITLE
Prompt System.ArgumentNullException: The parameter "unevaluatedValue" cannot be null.

### ISSUE THREAD

**Issue by 2505817596** [mono/Embeddinator-4000#783]
未经处理的异常:  System.ArgumentNullException: 参数“unevaluatedValue”不能为 null。
   在 Microsoft.Build.Shared.ErrorUtilities.VerifyThrowArgumentNull(Object parameter, String parameterName, String resourceName)
   在 Microsoft.Build.Construction.ProjectTaskElement.SetParameter(String name, String unevaluatedValue)
   在 Embeddinator.XamarinAndroidBuild.GenerateJavaStubsProject(List`1 assemblies, String outputDirectory)
   在 Embeddinator.Driver.WriteFiles()
   在 Embeddinator.Driver.Run()
   在 Embeddinator.CLI.Main(String[] args)

**Comment by chamons** [comment]
Thanks for the feedback.

It looks like you hit an exception while compiling an android project.

Could you please attach the project in question and build logs so we can look into th

**Comment by 2505817596** [comment]
> 
> 
> Thanks for the feedback.
> 
> It looks like you hit an exception while compiling an android project.
> 
> Could you please attach the project in question and build logs so we can look into th
[ClassLibrary1.zip](https://github.com/mono/Embeddinator-4000/files/5479380/ClassLibrary1.zip)

This is actually an Android class library project created, nothing changed

**Comment by chamons** [comment]
Can you please also post the tool invocation you used to try to bind that library?

**Comment by 2505817596** [comment]
> Can you please also post the tool invocation you used to try to bind that library?

Are you uploading my local Embeddinator-4000 tool?

**Comment by chamons** [comment]
No, what command did you use to build it from the command line?

**Comment by 2505817596** [comment]
> No, what command did you use to build it from the command line?

Embeddinator-4000.exe -gen=Java --platform=Android -out=foo ClassLibrary1.dll

**Comment by 2505817596** [comment]
> No, what command did you use to build it from the command line?

![1604447606374](https://user-images.githubusercontent.com/37748815/98053153-8d54ac00-1dfd-11eb-9ac5-8be37fdef662.jpg)
Hi, this problem was solved by adding a -c option, but now there is a new problem!

**Comment by datoml** [comment]
I had the same problem.
Try to run the command again with the -v option.
`Embeddinator-4000.exe -gen=Java --platform=Android -out=foo ClassLibrary1.dll -v`

The output should show that the Android Ndk registry key is missing.
What if have done was to add the missing key and pointed to the android ndk.
